instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import Comment from './Comment.js';
+import Authenticate from '../Authenticate/Authenticate.js';
+//import * as globalStyles from '../../globalStyles.js';
+import * as comments from './commentComponents.js';
+
+class CommentSection extends React.Component {
+    constructor(props) {
+        super(...arguments);
+        this.state = {
+            commentList: props.comments || [],
+            lastActivity: this.props.lastActivity,
+            likes: this.props.likes,
+            commentBody: ''
+        }
+    }
+    render() {
+        return (
+            <comments.Comments>
+                <comments.CommentsInteraction>
+                    <comments.CommentsInteractionImg
+                        src="img/interact-like.png"
+                        alt="like button"
+                        onClick={this.addLike}
+                    />
+                    <comments.CommentsInteractionImg
+                        src="img/interact-comment.png"
+                        alt="comment button"
+                    />
+                    <comments.CommentsLikesCount>{this.state.likes} likes</comments.CommentsLikesCount>
+                </comments.CommentsInteraction>
+                <div>
+                {
+                    this.state.commentList.map((comment, commentIndex) => (
+                        <Comment
+                            key={commentIndex}
```

Comment:
using only the index to set the key should always be our last resort . could concat the index with date.now() ? or
it is save to assume that all users on instagram usernames are going to be unique